src/scripts/main.js

```diff
@@ -3,6 +3,42 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+console.log(people);
 
-// write your code here
+const table = document.querySelector('tbody');
+
+for (const person of people) {
+  const someOne = document.createElement('tr');
+
+  const someOneName = document.createElement('td');
```

Comment:
```suggestion
  const nameColumn = document.createElement('td');
```
this naming will be more clear (everywhere)